src/react-justified-layout.js

```diff
@@ -94,10 +96,17 @@ class JustifiedLayout extends React.Component {
       <div style={{ ...Style, ...style, height: containerHeight, width: this.props.containerWidth }}>
         {map(elementLayout, ([element, layout]) => {
           const { height, left, top, width } = layout;
+
+          // Remove aspectRatio tag when rendering to prevent React errors:
+          let elementProps = element.props;
+          if (!preserveAspectRatioTag) {
+            delete elementProps.aspectRatio;
```

Comment:
Isn’t this mutating the props object?

Could we de-structure to avoid this?

`{ aspectRatio, ...elementProps }`

Reply:
Oh hey, you're right. 
```suggestion
            { aspectRatio, ...elementProps };
```

I'm guessing the destructuring there will be smart enough to overwrite the `elementProps` in the outer scope? I've never thought of doing that before.

Reply:
It doesn't sadly (give it a go in a `node` REPL and see what happens).

```js
let { aspectRation, ...rest } = element.props;
elementProps = rest;
```

Once you have made this change, add yourself as a contributor to package.json and I will merge and do a new release.

Thanks!